### PR TITLE
Unescape newlines in json test expectations.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by leonardoalt — DISMISSED** [review]

File: test/cmdlineTests/standard_ir_requested/output.json [hunk 1]
```diff
@@ -1 +1,87 @@
...
+ *                       WARNING                       *
+ *  Solidity to Yul compilation is still EXPERIMENTAL  *
+ *       It can result in LOSS OF FUNDS or worse       *
+ *                !USE AT YOUR OWN RISK!               *
+ *******************************************************/
+
+
+object \"C_6\" {
+\tcode {
+\t\tmstore(64, 128)
```
Comment (Marenz) [inline test/cmdlineTests/standard_ir_requested/output.json:11]: maybe also replace \t ?

**Comment by Marenz** [comment]
So whatever reads those files can handle both `\n` and actual new lines?

File: test/cmdlineTests/standard_ir_requested/output.json (hunk 1 above)
Comment (chriseth) [reply test/cmdlineTests/standard_ir_requested/output.json:11]: Yep, good idea!

**Comment by chriseth** [comment]
So the test will not be able to distinguish `\n` from actual newlines, but I guess this is fine because we can assume that the output is valid json.

**Comment by chriseth** [comment]
For tabs it might be different. Are literal tabs valid inside json strings?

File: test/cmdlineTests/standard_ir_requested/output.json (hunk 1 above)
Comment (chriseth) [reply test/cmdlineTests/standard_ir_requested/output.json:11]: Maybe it is better to not use tabs in the asm printer instead.

**Comment by chriseth** [comment]
Created issue for the tab part: https://github.com/ethereum/solidity/issues/7046

**Review by erak — COMMENTED** [review]
I think we could add even more newlines and spaces after e.g. `:` and `,` in the future, which would increase readability. But it's also fine for now.

Is this PR now blocked by https://github.com/ethereum/solidity/issues/7046?

**Comment by chriseth** [comment]
I would say not blocked.

**Review by Marenz — APPROVED** [review]

**Pull request merged**